src/index.js

```diff
@@ -1,4 +1,3 @@
 var Swarm = require('./swarm')
 
 exports = module.exports = Swarm
-exports.singleton = new Swarm()
```

Comment:
It was a old idea, where instead of passing the object around for Peer Routing, Distributed Record Store and Discovery mechanisms, one would simply do require('swarm').singleton. But eventually I never used it and just passed the swarm object along for making it more explicit. Good call on removing it! :)